docs/policies/policies.md

```diff
@@ -0,0 +1,480 @@
+        "issuanceDate": "2023-06-02T12:00:00Z",
+        "expirationDate": "2022-06-16T18:56:59Z",
+        "credentialSubject": {
+            "id": "<did:web:subject>",
+            "holderIdentifier": "BPN6789",
+            "items": [
+            "MembershipCredential",
+            "TraceabilityCredential"
+            ],
+            "contractTemplates": "https://public.catena-x.org/contracts/"
+        }
+    }
+
+#### Scenario 1
+
+Partner 1 wants to create an Access Policy, that Partner 2 can only receive the Contract offer if its BPN matches AND if it has the Traceability Credential (Traceability Contract Signed). If one of those is missing, Partner 2 won't receive a Contract Offer.
+
+##### Partner 1 - Access Policy Example
+
+    {
+        "@context": {
+            "odrl": "http://www.w3.org/ns/odrl/2/"
+        },
+        "@type": "PolicyDefinitionRequestDto",
+        "@id": "{{POLICY_ID}}",
+        "policy": {
+            "@type": "Policy",
+            "odrl:permission": [
+                {
+                    "odrl:action": "USE",
+                    "odrl:constraint": {
+                        "@type": "LogicalConstraint",
+                        "odrl:and": [
+                            {
+                                "@type": "Constraint",
+                                "odrl:leftOperand": "BusinessPartnerNumber",
+                                "odrl:operator": {
+                                    "@id": "odrl:eq"
+                                },
+                                "odrl:rightOperand": "{{BPN6789}}"
+                            },
+                            {
+                                "@type": "Constraint",
+                                "odrl:leftOperand": "FrameworkAgreement.traceability",
+                                "odrl:operator": {
+                                    "@id": "odrl:eq"
+                                },
+                                "odrl:rightOperand": "active"
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    }
+
+Be aware, the credential within the Summary Credential is called "TraceabilityCredential", but as the EDC does a mapping, the EDC Access Policy Entry needs to be named "FrameworkAgreement.traceability"!
+
+##### Desired Outcome
+
+Partner 2 receives the Contract Offer because the BPNs are matching and he owns the Traceability Credential.
+
+#### Scenario 2
+
+Partner 1 wants to create an Access Policy, that Partner 2 can receive the Contract Offer if its BPN matches. But a Contract Agreement should only be created if Partner 2 also signed the Traceability Framework Agreement. So in this case, Partner 2 should receive the Contract Offer in the first place, regardless if it signed the Traceability Framework Agreement. The signing of the Agreement should be checked at the time of contract negotiation.
+
+##### Partner 1 - Access Policy Example (Scenario 2)

```

Comment:
Why did you put `(Scenario 2)` at the end of the headline? (same for the coming headlines as well)

Reply:
Markdownlint complained here that there are headlines with the same text